src/Microsoft.DotNet.ProjectJsonMigration/Report/MigrationReport.cs

```diff
@@ -0,0 +1,31 @@
+// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Microsoft.DotNet.ProjectJsonMigration
+{
+    public class MigrationReport
+    {
+        public List<ProjectMigrationReport> ProjectMigrationReports { get; }
+
+        public int MigratedProjectsCount => ProjectMigrationReports.Count;
+
+        public int SucceededProjectsCount => ProjectMigrationReports.Count(p => !p.Errors.Any());
+
+        public int FailedProjectsCount => ProjectMigrationReports.Count(p => p.Errors.Any());
```

Comment:
We could make "Passed" and "Failed" properties on `ProjectMigrationReport`, which would make this a little more readable.

`public int FailedProjectsCount => ProjectMigrationReports.Count(p => p.Failed);`